### PR TITLE
Njray sharepoint ra

### PR DESCRIPTION
**Pull request by njray**
new deployment and validation steps, replacing earlier deploy steps

### CODE REVIEW

**Comment by bennage** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference-architectures/sharepoint/index.md](https://github.com/njray/architecture-center/blob/njray-sharepoint-ra/docs/reference-architectures/sharepoint/index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/sharepoint/index?branch=pr-en-us-176) | [Details](#user-content-docs%2Freference-architectures%2Fsharepoint%2Findex.md)

<a name="docs%2Freference-architectures%2Fsharepoint%2Findex.md"></a>
### [docs/reference-architectures/sharepoint/index.md](https://github.com/njray/architecture-center/blob/njray-sharepoint-ra/docs/reference-architectures/sharepoint/index.md)
 - **Line 328**: **[Warning]** Invalid file link:(~/reference-architectures/sharepoint/CLI).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C22%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201706222015287722-176%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference-architectures/sharepoint/index.md](https://github.com/njray/architecture-center/blob/njray-sharepoint-ra/docs/reference-architectures/sharepoint/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/sharepoint/index?branch=pr-en-us-176) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C22%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201706222018181906-176%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference-architectures/sharepoint/index.md](https://github.com/njray/architecture-center/blob/njray-sharepoint-ra/docs/reference-architectures/sharepoint/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/sharepoint/index?branch=pr-en-us-176) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C23%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201706230013024600-176%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference-architectures/sharepoint/index.md](https://github.com/njray/architecture-center/blob/njray-sharepoint-ra/docs/reference-architectures/sharepoint/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/sharepoint/index?branch=pr-en-us-176) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C23%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201706230058533758-176%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference-architectures/sharepoint/index.md [hunk 1]
```diff
@@ -111,143 +116,264 @@ Before configuring the SharePoint farm, make sure you have one Windows Server AD
...
+
+- [Azure Forum][azure-forum]
+
+- [TechNet Wiki: SharePoint: Community Best Practices][technet-wiki]
+
+- [SharePoint Microsoft Tech Community][sharepoint-community]
+
+## Solution deployment
+
+The deployment scripts for this reference architecture are available on [Github][github]. To run the PowerShell script that deploys this architecture, use the latest version of the Azure [command line interface][azure-cli] (CLI).
```
Comment (MikeWasson) [inline docs/reference-architectures/sharepoint/index.md]: I don’t understand the 2d sentence. PowerShell != CLI

File: docs/reference-architectures/sharepoint/index.md [hunk 2]
```diff
@@ -111,143 +116,264 @@ Before configuring the SharePoint farm, make sure you have one Windows Server AD
...
+| onprem         | Deploys a simulated on-premises network environment. This option does not connect to any existing on-premises networks. |
+| infrastructure | Deploys the SharePoint 2016 network infrastructure and jumpbox in Azure.                                                |
+| createvpn      | Deploys the virtual network gateway for both the SharePoint and on-premises networks and connects them.                 |
+| workload       | Deploys the SharePoint servers to the SharePoint network.                                                               |
+| security       | Deploys the network security group to the SharePoint network.                                                           |
+| all            | Deploys all the preceding deployments.                                                                                  |
+
+> [!NOTE]
+>  We recommend deploying this architecture incrementally in the order shown (infrastructure first, then applications). If you don’t have an on-premises network and want to deploy the entire reference architecture for testing and evaluation, specify **all** for the mode and note that process may take several hours.
+>
```
Comment (MikeWasson) [inline docs/reference-architectures/sharepoint/index.md]: These statements seem slightly contradictory:

- The first time, we recommend an incremental deployment so you can see what each deployment does. 
- We recommend deploying this architecture incrementally in the order shown

Sounds like we’re saying you should mostly use ‘incremental’ unless you want a test deployment?

File: docs/reference-architectures/sharepoint/index.md [hunk 3]
```diff
@@ -111,143 +116,264 @@ Before configuring the SharePoint farm, make sure you have one Windows Server AD
...
+
+### Scenario 2: Validate jumpbox access to VMs and check configuration settings
+
+From the jumpbox, you can create a remote connection and log on to any of the virtual machines and assets associated with this deployment. The following table shows various virtual machine names. The computer name shown in Azure Portal, such as **ad1**, is appended to **.contoso.local**, the SharePoint domain, to create the name you use to log on to a virtual machine from the jumpbox. For example, **ad1.contoso.local**.
+
+| Resource Name      | Purpose                                   | Resource Group        | VM Name                       |
+|--------------------|-------------------------------------------|-----------------------|-------------------------------|
+| Ra-sp2016-ad-vm1   | Active Directory + DNS box 1              | Ra-sp2016-network-rg  | Ad1.contoso.local             |
+| Ra-sp2016-ad-vm2   | Active Directory + DNS box 2              |                       | Ad2.contoso.local             |
```
Comment (MikeWasson) [inline docs/reference-architectures/sharepoint/index.md]: Missing RG in some rows

File: docs/reference-architectures/sharepoint/index.md [hunk 4]
```diff
@@ -111,143 +116,264 @@ Before configuring the SharePoint farm, make sure you have one Windows Server AD
...
-    - Click the **Purchase** button.
+6.  When prompted for your credentials, type **testuser** for the user and **AweS0me\@PW** for the password, then click **OK**. You will get a warning about the remote computer’s security certificate. This is normal. Click **Yes** to continue.
 
-3.	Check Azure portal notifications for a message that the first stage of the deployment is complete.
+7.  When the remote connection to the virtual machine opens, review the configuration and make changes using the administrative tools such as Server Manager.
 
-4.	Click the button below to begin the second stage of the deployment:
- 
-5.	When the link has opened in the Azure portal, enter the following values:
+#### To explore more, try the following:
```
Comment (MikeWasson) [inline docs/reference-architectures/sharepoint/index.md]: Is this section valuable? It does not seem very actionable. I'd remove it.

File: docs/reference-architectures/sharepoint/index.md [hunk 5]
```diff
@@ -111,143 +116,264 @@ Before configuring the SharePoint farm, make sure you have one Windows Server AD
...
+
+2.  In the list of resources, select **ra-adds-user-vm1**, the user virtual machine, then click **Connect** on the command bar atop the dashboard.
+
+3.  When prompted, click **Open** to open the Remote Desktop Protocol file (.rdp file). You will get a warning that the .rdp is from an unknown publisher. This is normal. In the Remote Desktop window, click **Connect** to continue.
+
+4.  When prompted for your credentials, type **\\onpremuser** for the user and **AweS0me\@PW** for the password, then click OK. If a warning message about authenticating the remove computer appears, click **Yes**.
+
+5.  When the remote connection to the user computer opens, open a browser and go to <http://portal.contoso.local>.
+
+6.  In the **Windows Security** box, log on to the SharePoint portal using **contoso.local\\testmuser** for the user name and **AweS0me\@PW** for the password.
```
Comment (MikeWasson) [inline docs/reference-architectures/sharepoint/index.md]: testmuser or testuser?

File: docs/reference-architectures/sharepoint/index.md [hunk 6]
```diff
@@ -1,105 +1,110 @@
...
 # Run a high availability SharePoint Server 2016 farm in Azure
+[!INCLUDE [header](../../_includes/header.md)]
+Contributors: Neil Hodgkinson, Bob Fox, Joe Davies, Paul Stork, Larry Brader
 
-This reference architecture shows a set of proven practices for setting up a high availability SharePoint Server 2016 farm using MinRole topology and SQL Server Always On availability groups on Azure. The SharePoint farm is deployed in a secured virtual network with no Internet-facing endpoint or presence.
+This reference architecture shows a set of proven practices for setting up a high availability SharePoint Server 2016 farm using MinRole topology and SQL Server Always On availability groups on Azure. The SharePoint farm is deployed in a secured intranet with no Internet-facing endpoint or presence.
 
-![](./images/sharepoint-ha.png)
+![[0]][0]
+Figure 1. Highly available SharePoint Server 2016 farm
```
Comment (MikeWasson) [inline docs/reference-architectures/sharepoint/index.md]: Is the diagram up to date?

File: docs/reference-architectures/sharepoint/index.md (hunk 1 above)
Comment (lbrader) [inline docs/reference-architectures/sharepoint/index.md]: Doesn't make sense.   It should be something like this  ".... use the latest version of PowerShell for Azure".

File: docs/reference-architectures/sharepoint/index.md (hunk 3 above)
Comment (lbrader) [inline docs/reference-architectures/sharepoint/index.md]: Yes...

File: docs/reference-architectures/sharepoint/index.md (hunk 4 above)
Comment (lbrader) [inline docs/reference-architectures/sharepoint/index.md]: makes sense... remove it

File: docs/reference-architectures/sharepoint/index.md (hunk 5 above)
Comment (lbrader) [inline docs/reference-architectures/sharepoint/index.md]: testuser

File: docs/reference-architectures/sharepoint/index.md (hunk 6 above)
Comment (lbrader) [inline docs/reference-architectures/sharepoint/index.md]: yes

**Comment by MikeWasson** [comment]
Incorporated the changes into #171

**Pull request closed without merging**